fjord/settings/base.py

```diff
@@ -29,7 +24,6 @@ def path(*dirs):
 ROOT_URLCONF = '%s.urls' % PROJECT_MODULE
 
 ADMINS = ()
-MANAGERS = ADMINS
```

Comment:
`MANAGERS` is only used by the `mail_managers` utility which we don't use. Thus there's no need to set this.